public/css/styles.css

```diff
@@ -67,41 +75,136 @@ h1 {
   position: relative;
   display: flex;
   justify-content: center;
-  max-width: 400px;
+  width: 100%;
 }
 
 .btn {
-  margin: 1rem;
+  /* margin-top: 1rem; */
   position: relative;
-  min-width: calc(200px - 1rem);
+  width: 100%;
+}
+
+button.btn.btn-info {
+  margin-right: 5px;
+}
+
+a.btn.btn-light {
+  margin-left: 10px;
+}
+
+.form-control {
+  width: 100% !important;
+  margin: 0 !important;
+}
+
+.navbar-collapse {
+  margin-top: 1rem;
 }
 
-/** NAVBAR **/
 .search-bar {
+  margin-bottom: 5px!important;
+}
+
+.nav-links {
+  margin-top: 0.25rem;
+}
+
+/* .nav {
```

Comment:
on second thought, i think we should remove all this commented out css. we can always come back to a previous PR if we need to reference old code.